dependencies.tsv

```diff
@@ -103,6 +103,6 @@ gopkg.in/macaroon.v1	git	ab3940c6c16510a850e1c2dd628b919f0f3f1464	2015-01-21T11:
 gopkg.in/mgo.v2	git	f2b6f6c918c452ad107eec89615f074e3bd80e33	2016-08-18T01:52:18Z
 gopkg.in/natefinch/lumberjack.v2	git	514cbda263a734ae8caac038dadf05f8f3f9f738	2016-01-25T11:17:49Z
 gopkg.in/natefinch/npipe.v2	git	c1b8fa8bdccecb0b8db834ee0b92fdbcfa606dd6	2016-06-21T03:49:01Z
-gopkg.in/retry.v1	git	c09f6b86ba4d5d2cf5bdf0665364aec9fd4815db	2016-10-25T18:14:30Z
+gopkg.in/retry.v1	git	8ac863816a3ab5f9a7fc69175e3d0bf3403d6b76	2017-05-31T08:20:38Z
```

Comment:
Perhaps best to point to retry master HEAD (01631078ef2fdce601e38cfe5f527fab24c9a6d2) ?